### ISSUE TITLE
Team page

### ISSUE THREAD

**Issue by vinaysethia27** [shreyashdholakia/squarecut.io-testing#3]
After creating a team name (Sixers), I wanted to add that team to MCL 2017 tournament list. But I couldn't add it. I tried to click on "+ participating tournament" but its not giving me any list of tournaments

![team page](https://cloud.githubusercontent.com/assets/25540491/22858107/06832fd2-f082-11e6-985c-f4806004e32a.PNG)

**Comment by shreyashdholakia** [comment]
@vinaysethia27 yes, it shows all the participating tournament team is part of. but would be good to have a link to tournaments page to join 👍
